modules/ROOT/pages/schema/graph-types/list-graph-types.adoc

```diff
@@ -0,0 +1,216 @@
+:description: Information about showing the graph type of a database.
+:page-role: cypher-25-only new-neo4j-2025.xx enterprise-edition
+= Show graph types
+
+The `SHOW CURRENT GRAPH TYPE` command allows you to show the graph type specification for the current database.
+
+[NOTE]
+Showing a graph type requires the link:{neo4j-docs-base-uri}/operations-manual/current/authentication-authorization/database-administration/#access-control-database-administration-constraints[`SHOW CONSTRAINTS` privilege].
+
+.Set graph type
+[source, cypher]
+----
+ALTER CURRENT GRAPH TYPE SET {
+   (p:Person => :Resident {name :: STRING, ssn :: INTEGER})
+     REQUIRE (p.name, p.ssn) IS KEY,
+   (:Pet => :Resident&Animal {insuranceNumber :: INTEGER IS KEY, healthCertificate :: STRING IS UNIQUE, name :: STRING}),
+   (:Robot => :Resident&Machine {application :: STRING NOT NULL, id :: INTEGER NOT NULL}),
+   (:City => {name :: STRING NOT NULL, population :: INTEGER}),
+   (:Resident)-[:LIVES_IN => {since :: ANY NOT NULL}]->(:City),
+   (:Company => {name :: STRING, address :: STRING IS UNIQUE}),
+   (:Person)-[:WORKS_FOR => {role :: STRING}]->(:Company),
+   CONSTRAINT company_name FOR (c:Company) REQUIRE c.name IS KEY,
+   CONSTRAINT animal_id FOR (a:Animal) REQUIRE a.id IS UNIQUE,
+   CONSTRAINT resident_address FOR (resident:Resident) REQUIRE resident.address :: STRING
+}
+----
+
+.Show the full graph type
+[source, cypher]
+----
+SHOW CURRENT GRAPH TYPE
+----
+
+.Result
```

Comment:
<img width="1365" height="1066" alt="image" src="https://github.com/user-attachments/assets/797b25c0-d680-425a-a547-461cf826eee3" />
not sure we need that much space on the end but sure :P

Reply:
(also applies to the next result)